SimPEG/Directives.py

```diff
@@ -527,114 +561,92 @@ class Update_IRLS(InversionDirective):
     @property
     def target(self):
         if getattr(self, '_target', None) is None:
-            self._target = self.survey.nD*0.5*self.chifact
+            nD = 0
+            for survey in self.survey:
+                nD += survey.nD
```

Comment:
we will need to think through generalizing this for a multi-physics inversion (where we will want each "physics" to reach its own target). For now, I think it is ok though

Reply:
Yes. I would vote for that. Need a different measure for each physics.

Reply:
Yea totally, that's a tricky one. We would probable need to store a list of targets or something.

Reply:
yeah, for now, lets add a note to the docs similar to my comment above